### DISCUSSION TITLE
Clean Desktop View

### DISCUSSION THREAD

**Discussion by cobbenterprises** [ttscoff/bunch-docs#290] [Q&A] [answered]
Is there a way to setup a bunch to clear the decks without closing the apps?  I see quit all and adding a color to the screen.  My use case is that when I have my macbook docked, I've got several apps running and in specific spots on the screens (pairing Bunch with Moom) but when I undock, I'd like them all to be minimized (not closed) and looking at my normal wallpaper since I'm probably going into more focused work rather than having all the other windows open.  

Thanks for the great tool and any help.

Jason

**Comment by ttscoff** [comment, marked as answer]
Well, there's `@@` to hide all visible apps, but it's not reliable for 
clearing the desktop completely as macOS requires one app to be focused 
when hiding everything. You can also specifically minimize apps by 
adding an underscore after the app name, e.g. `Ivory_`. You would have 
to add each app you want minimized, though, there's no "minimize open 
apps" command at present.

**Reply by cobbenterprises** [reply]
Got it.  Thank you very much.
